extension/android/benchmark/app/src/main/java/org/pytorch/minibench/BenchmarkActivity.java

```diff
@@ -47,7 +47,11 @@ protected void onCreate(Bundle savedInstanceState) {
     // TODO: Format the string with a parsable format
     Stats stats = new Stats();
 
+    // Record the time it takes to load the model
+    stats.loadStart = System.currentTimeMillis();
```

Comment:
Let's use https://docs.oracle.com/javase/7/docs/api/java/lang/System.html#nanoTime() (for all places as well)